Gemfile.lock

```diff
@@ -722,7 +728,7 @@ GEM
       thor (~> 1.0)
     rainbow (3.1.1)
     raindrops (0.20.0)
-    rake (12.3.3)
+    rake (13.1.0)
```

Comment:
yes, it turns out to be intentional! `llhttp-ffi` depends on it: https://github.com/bryanp/llhttp/blob/b4cf901ef8b4a8946fb6d2b18f6624c6e2708107/ffi/llhttp-ffi.gemspec#L25

I'm spinning up an adhoc now to see if that could catch some problems. I'm also open to shipping the rake upgrade ahead of this one.

Reply:
Ahh, I missed that. Spinning up an adhoc sounds great! I think given that we don't have a direct top-level dependency on rake in the Gemfile, it's probably fine to include the rake upgrade as part of this rather than splitting it out into its own PR.

Reply:
adhoc is up: https://adhoc-pin-http-5-for-statsig.cdn-code.org/students